locator/tablet_replication_strategy.hh

```diff
@@ -50,6 +50,12 @@ public:
     /// otherwise, cur_tablets is a copy of the current tablet_map.
     /// Runs under group0 guard.
     virtual future<tablet_map> reallocate_tablets(schema_ptr, token_metadata_ptr, tablet_map cur_tablets) const = 0;
+
+    /// Returns desired replication factor in a given DC.
+    /// Note that individual tablets may lag behind desired replication factor in their
+    /// current replica list, as replication factor changes invole table rebuilding transitions
+    /// which are not instantenous.
```

Comment:
instantaneous, don't get @tchaikov's spelling bot angry at you

Reply:
If it is the desired replication factor rather than the actual one, I suggest to reflect it in the method name, e.g. `get_desired_replication_factor`.

Reply:
But there's no other kind of replication factor, so it's redundant to add extra qualifiers. 

This doesn't add a new method, only lifts it to the interface.

Reply:
Maybe drop the word "desired" from the comment then.
It's confusing IMO.

Reply:
Ok, I can drop it.

Reply:
Still not dropped in latest version.
Also, there's a typo in the comment: s/invole/involve/